### PR TITLE
Add billing_model field to cluster type

### PR DESCRIPTION
**Pull request by zgalor**
cc @rawsyntax 
@tzvatot @igoihman @jhernand PTAL

### CODE REVIEW

**Comment by nimrodshn** [comment]
LGTM :100:

File: model/clusters_mgmt/v1/cluster_type.model [hunk 1]
```diff
@@ -190,4 +190,7 @@ class Cluster {
 
 	// Channel group to be used for upgrading the cluster. Valid values: stable (default), fast, candidate, nightly.
 	UpgradeChannelGroup String
+
+	// Billing model for cluster resources. Valid values: standard (default), marketplace.
+	BillingModel String
```
Comment (jhernand) [inline model/clusters_mgmt/v1/cluster_type.model:195]: Is there any reason to not use an _enum_ type for this?

Comment (zgalor) [reply model/clusters_mgmt/v1/cluster_type.model:195]: No. I'll fix it

**Comment by zgalor** [comment]
@jhernand rePTAL

**Review by jhernand — APPROVED** [review]
Looks good to me. Thanks @zgalor.

File: model/clusters_mgmt/v1/billing_model_type.model [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+limitations under the License.
+*/
+
+// Billing model for cluster resources.
+enum BillingModel {
+	// Standard. This is the default billing model
+	Standard
+
+	// Red Hat Marketplace billing model.
+	Marketplace
```
Comment (petli-openshift) [inline model/clusters_mgmt/v1/billing_model_type.model:23]: both should be lower-case. sry for the misleading instructions in the card. We've changed it after the initial design.

@rawsyntax I think we should update our code to be case-insensitive. Now it rejects `Standard`,
```
{
  "id": "8",
  "kind": "Error",
  "href": "/api/accounts_mgmt/v1/errors/8",
  "code": "ACCT-MGMT-8",
  "reason": "BillingModel is invalid: Standard",
  "operation_id": "1oCq468gjV68YkeVoknLybqyjSm"
}
```

Comment (zgalor) [reply model/clusters_mgmt/v1/billing_model_type.model:23]: @petli-openshift ocm api only supports lower case inputs.
https://github.com/openshift-online/ocm-api-metamodel project generates the SDK and the openapi yaml file and translates these camel case values to snake case.

**Pull request merged**